x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -220,6 +220,33 @@ export class Simulator {
     );
   }
 
+  /**
+   * For a render of the Enzyme wrapper (may help update if other concerns have effected the render). May be useful for debugging.
+   */
+  public updateWrapper() {
+    return this.wrapper.update();
+  }
+
+  /**
+   * Dump all contents of the outer ReactWrapper (to be `console.log`ged as appropriate)
+   */
+  public debugWrapper() {
+    return this.wrapper.debug();
+  }
+
+  /**
+   * Return an Enzyme ReactWrapper that includes the Related Events host button for a given process node
+   *
+   * @param entityID The entity ID of the proocess node to select in
+   */
+  public processNodeRelatedEventButton(entityID: string): ReactWrapper {
+    return this.processNodeElements({ entityID }).findWhere(
+      (wrapper) =>
+        typeof wrapper.type() === 'string' &&
```

Comment:
can you add a comment here saying something like 'filter out React components'?

Reply:
https://github.com/elastic/kibana/pull/74411/commits/a48a10a6244e3ea35eda3de095fddb427e645c91#diff-d5aefb42e7a51c722db06ebb62bc1a47R239